gradle/dependencies.gradle

```diff
@@ -24,6 +24,7 @@ ext {
     kotlin            : "1.3.72",
     coroutines        : "1.3.0",
     springboot        : "2.3.1.RELEASE",
+    junit5            : "5.7.0-M1",
```

Comment:
Do we need a milestone or current stable is enough?

Reply:
Yeah this milestone fixed a spock issue